src/lava/proc/dense/models.py

```diff
@@ -23,22 +23,23 @@ class PyDenseModelFloat(PyLoihiProcessModel):
     fixed point implementation.
     """
     s_in: PyInPort = LavaPyType(PyInPort.VEC_DENSE, bool, precision=1)
-    a_out: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE,np.float)
-    a_buff: np.ndarray = LavaPyType(np.ndarray,np.float)
-    weights: np.ndarray = LavaPyType(np.ndarray,np.float)
-    weight_exp: float = LavaPyType(float,np.float)
-    num_weight_bits: float = LavaPyType(float,np.float)
-    sign_mode: float = LavaPyType(float,np.float)
+    a_out: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, np.float)
```

Comment:
Doesn't this lead to a deprecation warning that you are supposed to use float instead of np.float?

Reply:
Changed all instances of np.float to float.